fix(APM): Removed Explorer option from the all capabilities section (#13828)
newrelic/docs-website: src/content/docs/apm/transactions/transaction-traces/introduction-transaction-traces.mdx

To view transaction traces:

> 1. Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities)** > **APM & Services**. Then, click your app and, under the **Monitor** section, click **Transactions**.
> 2. In the **Transaction traces** section, click a transaction trace to view [additional details](/docs/apm/transactions/transaction-traces/transaction-traces-summary-page).

Reviewer: ```suggestion
1. Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > APM & Services**. Select an app, then under the **Monitor** section click **Transactions**.
2. In the **Transaction traces** section, click a transaction trace to view [additional details](/docs/apm/transactions/transaction-traces/transaction-traces-summary-page).
```
